Add TTFB guideline thresholds (#8414)
GoogleChrome/web.dev: src/site/content/en/metrics/ttfb/index.md

### What is a good TTFB score?

> Due to the wide variation of network and application backend stacks, an arbitrary number can't be placed on what consists of a "good" TTFB score. Because TTFB precedes [user-centric metrics](/user-centric-performance-metrics/) such as [First Contentful Paint (FCP)](/fcp/) and [Largest Contentful Paint (LCP)](/lcp/), it's recommended that your server responds to navigation requests quickly enough so that the **75th percentile** of users experience an [FCP within the "good" threshold](/fcp/#what-is-a-good-fcp-score). However, as a rough guide, most sites should strive to have Time To First Byte of **0.8 seconds** or less.

Author: Yeah I struggled with this but think it's still valid that any thresholds we give here are somewhat arbitrary, because they are not [user-centric](https://web.dev/user-centric-performance-metrics/) nor based on a [rigorous, well-defined, process](https://web.dev/defining-core-web-vitals-thresholds/).

What about removing the "can't be placed" wording to somewhat avoid the "we can't recommend, but here's one anyway"?

```suggestion
Due to the wide variation of network and application backend stacks, a "good" TTFB score—and the impact of that—can vary. Because TTFB precedes [user-centric metrics](/user-centric-performance-metrics/) such as [First Contentful Paint (FCP)](/fcp/) and [Largest Contentful Paint (LCP)](/lcp/), it's recommended that your server responds to navigation requests quickly enough so that the **75th percentile** of users experience an [FCP within the "good" threshold](/fcp/#what-is-a-good-fcp-score). However, as a rough guide, most sites should strive to have Time To First Byte of **0.8 seconds** or less.
```

> Due to the wide variation of network and application backend stacks, an arbitrary number can't be placed on what consists of a "good" TTFB score. Because TTFB precedes [user-centric metrics](/user-centric-performance-metrics/) such as [First Contentful Paint (FCP)](/fcp/) and [Largest Contentful Paint (LCP)](/lcp/), it's recommended that your server responds to navigation requests quickly enough so that the **75th percentile** of users experience an [FCP within the "good" threshold](/fcp/#what-is-a-good-fcp-score). However, as a rough guide, most sites should strive to have Time To First Byte of **0.8 seconds** or less.

Reviewer: > because they are not user-centric nor based on a rigorous, well-defined, process

I think we can remove the entire first sentence and lead with that.

> Due to the wide variation of network and application backend stacks, an arbitrary number can't be placed on what consists of a "good" TTFB score. Because TTFB precedes [user-centric metrics](/user-centric-performance-metrics/) such as [First Contentful Paint (FCP)](/fcp/) and [Largest Contentful Paint (LCP)](/lcp/), it's recommended that your server responds to navigation requests quickly enough so that the **75th percentile** of users experience an [FCP within the "good" threshold](/fcp/#what-is-a-good-fcp-score). However, as a rough guide, most sites should strive to have Time To First Byte of **0.8 seconds** or less.

Reviewer: LGTM

<figure>
  <picture>
    <source
      srcset="{{ "image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/ILJ1xKjzVisqOPPyHYVA.svg" | imgix }}"
      media="(min-width: 640px)"
      width="800"
      height="200">
    {%
      Img
        src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/eNXaxPi9NdUVSTDRJFkV.svg",
        alt="Good TTFB values are 0.8 seconds or less, poor values are greater than 1.8 seconds, and anything in between needs improvement",
        width="640",
        height="480"
    %}
  </picture>
</figure>

{% Aside %}

>   The above thresholds are guideline thresholds. Sites can still receive good Core Web Vitals with TTFB timings that exceed this depending on the network and application stack.

Author: LGTM but let's add the link:

```suggestion
  TTFB is not a [Core Web Vitals](/vitals) metric, so it's not absolutely necessary that sites meet the "good" TTFB threshold, provided that it doesn't impede their ability to score well on the metrics that matter.
```
